frontend/app/conversations/[id].tsx

```
@@ -616,10 +661,14 @@ const styles = StyleSheet.create({
   messagesList: {
     flex: 1,
   },
+  composerContainer: {
+    flexShrink: 0,
+    backgroundColor: colors.surface,
+  },
```

ESLint

[error] Unused style detected: styles.composerContainer

(react-native/no-unused-styles)